src/test/java/com/github/discvrseq/walkers/VcfToLuceneIndexerIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package com.github.discvrseq.walkers;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Arrays;
+
+public class VcfToLuceneIndexerIntegrationTest extends BaseIntegrationTest {
+
+    // Case where all IF are valid
+    @Test
+    public void doBasicTest() throws Exception {
+        ensureVcfIndex(getInputVcf());
+
+        IntegrationTestSpec spec = new IntegrationTestSpec(
```

Comment:
Does this actually work?? At least for me, IntegrationTestSpec with "-O %s" will create a temp file (not folder), and then this blows up downstream?